sniplates/templates/sniplates/bootstrap.html

```diff
@@ -0,0 +1,23 @@
+{% block textbox %}
+<div class="form-group">
+    {% if col1 %}
+    <div class="col-md-{{col1}}">
+    {% endif %}
+        <label for="id_{{ field.name }}">{{ field.label }}</label>
+    {% if col1 %}
+    </div>
+    {% endif %}
+    {% if col2 %}
+    <div class="col-md-{{col2}}">
+    {% endif %}
+        <input class="form-control" value="{% if value %}{{ value }}{% else %}{{ field.value }}{% endif %}">
```

Comment:
Just use :

    {{ value|default:"" }}

If you want to override it, pass it as an arg to the tag... otherwise it's already set from the field.